scripts/bazel_pull.sh

```diff
@@ -9,6 +9,13 @@ for cmd in base64 diff git grep sed xargs; do
     which "$cmd" >/dev/null
 done
 
+git submodule foreach --recursive -q pwd            \
+| xargs -rI{} realpath --relative-to="$(pwd)" {}    \
+| sed 's/\/*$//'                                    \
+| grep -e'cris-'{base,xchg}'$'                      \
```

Comment:
Why they are different?

Reply:
Because you want to pull the branch....
Other repos doesn't have dynamic submodules.

Reply:
Do we need to push to the remote then? If not maybe we should not do that because that will be different than what engineers see. What do you think?

Reply:
No.

This is to preserve the existing behavior.
Ideally we shouldn't do this, but there's no bot to update submodules yet, so I have to keep what bazel's doing.